airbyte-ci/connectors/base_images/base_images/root_images.py

```diff
@@ -15,5 +15,5 @@
     registry="docker.io",
     repository="python",
     tag="3.9.19-slim-bookworm",
-    sha="b92e6f45b58d9cafacc38563e946f8d249d850db862cbbd8befcf7f49eef8209",
```

Comment:
This was the mistake, this sha is the one of the ARM64 image, not the sha of the multi platform one.